.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+      - unshitify # Add your feature branch here
+  pull_request:
+    branches:
+      - main
+      - master # Add master as a target branch
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)